spec/neo4j/driver/dev_manual_examples_spec.rb

```diff
@@ -34,8 +34,8 @@
 
     context 'Example 2.3. Custom Address Resolver' do
```

Comment:
let's add something like
```
      let(:uri) { 'neo4j://nonexisting' }
```
so the spec is effective regardless of the global scheme.